docker/localsettings.py

```diff
@@ -1,4 +1,4 @@
-####### Configuration for CommCareHQ Running in docker #######
+"""####### Configuration for CommCareHQ Running in docker #######"""
```

Comment:
Interesting solution to that lint error. Would it also work to prefix the line with `# `?
```suggestion
# ####### Configuration for CommCareHQ Running in docker #######
```
Not saying you should change it, just trying to think of another solution.

Reply:
Yeah it will work with `#  `, I tried that too but did not like the extra space in the beginning 😅